zyte_api/__main__.py

```diff
@@ -53,15 +57,32 @@ async def run(queries, out, n_conn, stop_on_errors, api_url,
     logger.info(f"\nException types:\n{client.agg_stats.exception_types.most_common()}")
 
 
+def _guess_intype(file_name, lines):
```

Comment:
Could you please move it to some utils module, and add a few tests?

Reply:
Do you mean “add a few tests” as in “add a few _more_ tests beyond those covering `test_guess_intype`”? Maybe tests for `read_input` or for the command-line interface entry point?